bundles/framework/myplaces3/service/MyPlacesService.js

```diff
@@ -406,8 +391,8 @@ Oskari.clazz.define('Oskari.mapframework.bundle.myplaces3.service.MyPlacesServic
             place.setLink(Oskari.util.sanitize(properties.link));
             place.setImageLink(Oskari.util.sanitize(properties.image_url));
             place.setCategoryId(properties.category_id);
-            place.setCreateDate(this._formatDate(properties.created));
-            place.setUpdateDate(this._formatDate(properties.updated));
+            place.setCreateDate(properties.created, false);
```

Comment:
There's no boolean value for the setters: https://github.com/oskariorg/oskari-frontend/blob/develop/bundles/framework/myplaces3/model/MyPlace.js#L162